moped-editor/src/utils/activityLogFormatters/mopedComponentsActivity.js

```diff
@@ -0,0 +1,60 @@
+import RoomOutlinedIcon from "@material-ui/icons/RoomOutlined";
+import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
+
+export const formatComponentsActivity = (change, componentList) => {
+  const entryMap = ProjectActivityLogTableMaps["moped_proj_components"];
+
+  const changeIcon = <RoomOutlinedIcon />;
+  const component =
+    componentList[change.record_data.event.data.new.component_id];
+  const componentText = {
+    text: component,
+    style: "boldText",
+  };
+
+  // add a new component
+  if (change.description.length === 0) {
+    return {
+      changeIcon,
+      changeText: [{ text: "Added a component: ", style: null }, componentText],
```

Comment:
Noticed a `:` here and in the remove message that I didn't see anywhere else in the log. Gotta admit that I didn't look at every change type though!